metadata/gecko.ttl

```diff
@@ -0,0 +1,25 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms: <http://purl.org/dc/terms/>.
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+<>
+  rdf:type owl:Ontology ;
+  dcterms:title "Genomics Cohorts Knowledge Ontology" ;
+  dcterms:description "An ontology to represent genomics cohort attributes. The GECKO is maintained by the CINECA project, https://www.cineca-project.eu, and standardises attributes commonly used for genomics cohort description as well as individual-level data items. A series of tools is being developed to enable automated generation of harmonised data files based on a JSON schema mapping file. For more information please contact info@cineca-project.eu" ;
+  dcterms:license <https://creativecommons.org/licenses/by/4.0/> ;
+  dcterms:rights "You are free to:
```

Comment:
I think that @mcourtot would prefer to just use the license, and include rights.

Reply:
I'm not quite sure what to put into rights here - if you have text you can reuse please do. I'm still not sure why we need both as per https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1019#issuecomment-595836706, but not a gigantic issue.

Reply:
Ok, then I prefer to just use license and remove rights, for now.

Reply:
seems the OBO dashboard will warn if rights is not in, so suggest keep?

Reply:
No, the dashboard only requires license. Your call!